### DISCUSSION TITLE
Fake note pre-empt mod

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#13157] [Ideas]
in all modes, all notes will have 2 fake objects appear a randomized distance from(but not at) the location where the object is supposed to be hit

then, all fake objects move toward the location of where you're supposed to hit the object near the time you're supposed to hit the object

to help explain what i'm talking about, here's a picture:
![Untitled](https://user-images.githubusercontent.com/44514667/93735313-2c04b080-fba2-11ea-923e-74ff35db860b.png)

**Comment by enoslayd** [comment]
similar to #8835

**Comment by enoslayd** [comment]
edit: confusing

**Comment by iSimpFor** [comment]
it's supposed to be notes moving in an unintended way to the location you're supposed to hit the note so that you have the read and adapt continuously

**Comment by mk56-spn** [comment]
Going to take this one up myself since i want to see it done and done well, despite the apparent simplicity i feel a good implementation is going to be a considerable undertaking, many things need to go into making this not feel cheaply made :

- A good methodology for placing these fake notes in believable places. ( probably going to require some basic pattern recognition based on angles and new combos, which should be done anyways for future improvements to random ... ) 

- Some sort of state cloning to transpose mod effects onto the fake circles ( Hidden, traceable ) 

- and a bunch more issues i likely haven't thought of yet. with that said it'll likely be a while until this is finished if it is at all, considering i have a decent few tasks that are much higher priority
